sherpa/astro/tests/test_astro_plot.py

```diff
@@ -1,5 +1,6 @@
 #
-#  Copyright (C) 2007, 2015, 2018, 2019, 2020, 2021  Smithsonian Astrophysical Observatory
+#  Copyright (C) 2007, 2015, 2018, 2019, 2020, 2021
```

Comment:
At some point, we should really make a post-commit hook that takes care of this stupid updating of year automatically!